README.md

```diff
@@ -33,9 +33,9 @@ Superbuild
 The `robotology-superbuild` is an infrastructure to simplify development and use of **open source research software** developed at the **[Italian Institute of Technology](https://iit.it/)**, in particular as part of the **[iCub project](https://icub.iit.it/)**. 
 
 ### Profiles and Optional Dependencies
-As a huge number of software projects are contained in the `robotology-superbuild`, and a tipical user is only interested in some of them, there are several **options** to instruct the superbuild on which packages should be built and which one should not be built. In particular, the robotology-superbuild is divided in different **profiles**, that specify the specific subset of robotology packages to build. You can read more on the available **profiles** and how to enabled them in the [`doc/profiles.md` documentation](doc/profiles.md).
+As a huge number of software projects are contained in the `robotology-superbuild`, and a tipical user is only interested in some of them, there are several **options** to instruct the superbuild on which packages should be built and which one should not be built. In particular, the robotology-superbuild is divided in different **profiles**, that specify the specific subset of robotology packages to build. You can read more on the available **profiles** and how to enabled them in the [`doc/cmake-options.md#profile-specific-documentation` documentation](doc/cmake-options.md#profile-specific-documentation).
```

Comment:
Just a nitpick, I don't know if the ending "-documentation documentation" in [x](url) is readable. If it's just me being pedantic, please ignore :D

Reply:
Fixed!